spanner/api/Program.cs

```diff
@@ -0,0 +1,989 @@
+                     AlbumId      INT64 NOT NULL,
+                     AlbumTitle   STRING(MAX)
+                 ) PRIMARY KEY (SingerId, AlbumId),
+                 INTERLEAVE IN PARENT Singers ON DELETE CASCADE";
+                // Make the request.
+                cmd = connection.CreateDdlCommand(createTableStatement);
+                await cmd.ExecuteNonQueryAsync();
+            }
+            return;
+            // [END create_database]
+        }
+
+        public static async Task AddIndexAsync(
+            string projectId, string instanceId, string databaseId)
+        {
+            // [START create_index]
+            // Initialize request argument(s).
+            string connectionString =
+                $"Data Source=projects/{projectId}/instances/"
+                + $"{instanceId}/databases/{databaseId}";
+            string createStatement =
+                $"CREATE INDEX AlbumsByAlbumTitle ON Albums(AlbumTitle)";
+            // Make the request.
+            using (var connection = new SpannerConnection(connectionString))
+            {
+                var createCmd = connection.CreateDdlCommand(createStatement);
+                await createCmd.ExecuteNonQueryAsync();
+            }
+            Console.WriteLine("Added the AlbumsByAlbumTitle index.");
+            return;
+            // [END create_index]
+        }
+
+        public static async Task AddStoringIndexAsync(
+    string projectId, string instanceId, string databaseId)
+        {
+            // [START create_storing_index]
+            // Initialize request argument(s).
+            string connectionString =
+                $"Data Source=projects/{projectId}/instances/"
+                + $"{instanceId}/databases/{databaseId}";
+            string createStatement =
+                $"CREATE INDEX AlbumsByAlbumTitle2 ON Albums(AlbumTitle) "
+                + "STORING (MarketingBudget)";
+            // Make the request.
+            using (var connection = new SpannerConnection(connectionString))
+            {
+                var createCmd = connection.CreateDdlCommand(createStatement);
+                await createCmd.ExecuteNonQueryAsync();
+            }
+            return;
+            // [END create_storing_index]
+        }
+
+        public static async Task AddColumnAsync(
+    string projectId, string instanceId, string databaseId)
+        {
+            // [START add_column]
+            // Initialize request argument(s).
+            string connectionString =
+                $"Data Source=projects/{projectId}/instances/"
+                + $"{instanceId}/databases/{databaseId}";
+            string createStatement =
+                $"ALTER TABLE Albums ADD COLUMN MarketingBudget INT64";
+            // Make the request.
+            using (var connection = new SpannerConnection(connectionString))
+            {
+                var createCmd = connection.CreateDdlCommand(createStatement);

```

Comment:
shouldn't this local be called "updateCmd" instead of "create" since its an ALTER TABLE?